source/_components/camera.local_file.markdown

```diff
@@ -35,3 +35,11 @@ Configuration variables:
 The given `file_path` must be an existing file because the camera platform setup make a readable check on it.
 </p>
```

Comment:
This paragraph is not really accurate (and apparently wasn't before either). The file does not need to exist, there's just a warning if it doesn't. Maybe remove it?

Reply:
Agreed